src/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.vbproj

```diff
@@ -35,180 +27,116 @@
     <Compile Include="..\Common\ManagedCodeMarkers.vb">
       <Link>ManagedCodeMarkers.vb</Link>
     </Compile>
-    <Compile Include="ApplicationDesigner\ApplicationDesignerEditorFactory.vb" />
-    <Compile Include="ApplicationDesigner\ApplicationDesignerLoader.vb" />
-    <Compile Include="ApplicationDesigner\ApplicationDesignerPanel.vb">
+    <Compile Update="ApplicationDesigner\ApplicationDesignerPanel.vb">
       <SubType>Component</SubType>
     </Compile>
-    <Compile Include="ApplicationDesigner\ApplicationDesignerRootComponent.vb">
+    <Compile Update="ApplicationDesigner\ApplicationDesignerRootComponent.vb">
       <SubType>Component</SubType>
     </Compile>
-    <Compile Include="ApplicationDesigner\ApplicationDesignerRootDesigner.vb" />
-    <Compile Include="ApplicationDesigner\ApplicationDesignerView.vb">
+    <Compile Update="ApplicationDesigner\ApplicationDesignerView.vb">
       <SubType>Component</SubType>
     </Compile>
-    <Compile Include="ApplicationDesigner\ApplicationDesignerWindowPane.vb" />
-    <Compile Include="ApplicationDesigner\ApplicationDesignerWindowPaneControl.vb">
+    <Compile Update="ApplicationDesigner\ApplicationDesignerWindowPaneControl.vb">
       <SubType>UserControl</SubType>
     </Compile>
-    <Compile Include="ApplicationDesigner\CmdTargetHelper.vb" />
-    <Compile Include="ApplicationDesigner\CustomViewProvider.vb" />
-    <Compile Include="ApplicationDesigner\DeferrableWindowPaneProviderService.vb" />
-    <Compile Include="ApplicationDesigner\DesignerTabButton.vb">
+    <Compile Update="ApplicationDesigner\DesignerTabButton.vb">
       <SubType>Component</SubType>
     </Compile>
-    <Compile Include="ApplicationDesigner\DesignerTabControl.vb">
+    <Compile Update="ApplicationDesigner\DesignerTabControl.vb">
       <SubType>Component</SubType>
     </Compile>
-    <Compile Include="ApplicationDesigner\DesignerTabControlRenderer.vb" />
-    <Compile Include="ApplicationDesigner\ErrorControlCustomViewProvider.vb" />
-    <Compile Include="ApplicationDesigner\IVsEditWindowNotify.vb" />
-    <Compile Include="ApplicationDesigner\PropertyPageInfo.vb" />
-    <Compile Include="ApplicationDesigner\PropertyPageSite.vb" />
-    <Compile Include="ApplicationDesigner\SpecialFileCustomView.Designer.vb">
-      <DependentUpon>SpecialFileCustomView.vb</DependentUpon>
+    <Compile Update="ApplicationDesigner\SpecialFileCustomView.Designer.vb">
```

Comment:
Not sure how Compile Update works different from Include

Also, `EnableDefaultItems` enables globbing? Hence enables us to remove the redundant includes?